meson.build

```diff
@@ -107,6 +110,10 @@ installed_test_bindir = join_paths(libexecdir, 'installed-tests', meson.project_
 installed_test_datadir = join_paths(datadir, 'installed-tests', meson.project_name())
 
 gio = dependency('gio-2.0', version : '>= 2.45.8')
```

Comment:
how does this get resolved for a Windows build?  Don't you need a gio dll for it?

Reply:
I guess `mingw64-glib2`?